enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/provider/model/request/EvaluateClientRequest.java

```diff
@@ -22,13 +22,16 @@
 import com.wultra.core.annotations.PublicApi;
 import lombok.*;
 
+import java.util.List;
+
 /**
  * Request object for {@link OnboardingProvider#evaluateClient(EvaluateClientRequest)}.
  *
  * @author Lubos Racansky, lubos.racansky@wultra.com
  */
 @Builder
 @Getter
+@Setter
```

Comment:
Do we really need the setter? Until now, the class has been immutable. (Or wannabe at least).

Reply:
Ok, setter is not used now. I wanted to keep the condition with `config.isSendingExtractedDataEnabled()` in the main method so the logic when extracted data are included is clear.